iothub/service/src/Exceptions/IotHubServiceErrorCode.cs

```diff
@@ -188,10 +196,20 @@ public enum IotHubServiceErrorCode
 
         /// <summary>
         /// IoT hub throttling limits have been exceeded for the requested operation.
+        /// <para>
         /// For more information, <see href="https://aka.ms/iothubthrottling">IoT hub quotas and throttling</see>.
+        /// </para>
         /// </summary>
         ThrottlingException = 429001,
 
+        /// <summary>
+        /// IoT hub throttling limits have been exceeded for the requested operation.
+        /// <para>
+        /// For more information, <see href="https://aka.ms/iothubthrottling">IoT hub quotas and throttling</see>.
+        /// </para>
+        /// </summary>
+        ThrottlingBacklogTimeout = 429003,
```

Comment:
Oh wow, service has multiple error subcodes for throttling! I wish they had added some more context in their swagger files.

Reply:
Totally